command/smartContract.go

```diff
@@ -60,7 +60,7 @@ func (cmd *Command) generateSmartContractToken() {
 		cmd.log.Error("Failed to generate smart contract token", "err", err)
```

Comment:
@Allen-Cherian  While running the command `generatesct` with an invalid DID, I am getting the following error message in the terminal:

```
2024-10-14T16:50:51.121+0530 [ERROR] Main: Failed to generate smart contract token: err=<nil>
``` 

The actual error message which suggests that the input DID doesn't exists doesn't appear. This error message, however, appears on the node logs, which is expected.

In the highlighted part, replace `err` with `basicResponse.Message` to get the error message on the terminal from the command is run.